numba_cuda/numba/cuda/cudadrv/driver.py

```diff
@@ -2190,9 +2185,8 @@ def callback(stream, status, future):
         return future
 
 
-class Event(object):
-    def __init__(self, context, handle, finalizer=None):
-        self.context = context
+class Event:
+    def __init__(self, handle, finalizer=None):
```

Comment:
+1 on this being something that has been used with the `context` argument @gmarkall